spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/DenseTensor.scala

```diff
@@ -1434,30 +1434,64 @@ private[tensor] class DenseTensor[@specialized T: ClassTag](
   }
 
   override def toString(): String = {
+    val foldThreshold = System.getProperty("bigdl.tensor.fold", "1000").toInt
```

Comment:
Add this new property to the document page.

https://github.com/intel-analytics/BigDL/blob/master/docs/docs/ScalaUserGuide/configuration.md